packages/actor-init-sparql/lib/ActorInitSparql-browser.ts

```diff
@@ -107,6 +108,9 @@ export class ActorInitSparql extends ActorInit implements IActorInitSparqlArgs {
       context[KEY_CONTEXT_QUERYFORMAT] = context.queryFormat;
       delete context.queryFormat;
     }
+    if (Array.isArray(context[KEY_CONTEXT_SOURCES])) {
+      context[KEY_CONTEXT_SOURCES] = AsyncReiterableArray.forFixedData(context[KEY_CONTEXT_SOURCES]);
```

Comment:
I think the standard is to use `from`, i.e., `fromFixedData`, when doing conversions from one datatype to another like this. (Node has `Array.from()` for example)

Reply:
Sure, I can change that :-)